var/spack/repos/builtin/packages/python/package.py

```diff
@@ -475,21 +485,46 @@ def write_easy_install_pth(self, exts):
                         "sys.__egginsert = p+len(new)\n")
 
     def activate(self, ext_pkg, **args):
+        path_view = args.get("path_view", "")
+
         ignore = self.python_ignore(ext_pkg, args)
         args.update(ignore=ignore)
 
         super(Python, self).activate(ext_pkg, **args)
 
-        exts = spack.store.layout.extension_map(self.spec)
+        # if we are activating in a filesystem view, we need to remove
+        # the link to the main easy_install.pth file prior to writing
+        # a custom one
+        if path_view:
```

Comment:
> it only ignores easy_install.pth files under bin/

When you say that do you mean this pattern:

```r'bin/easy_install[^/]*$',```

Because a few lines above it I also see:

```r'site-packages/easy-install\.pth$'```

So as far as I can tell, this should already be taken care of (and for example I see the develop branch exclude easy_install.pth if I activate py-pytz). If you see that happen then I should probably investigate that.

> the easy_install.pth file under site-packages was linked (which I think is correct behavior for globally activated extensions as it was before).

As of now ```Python.write_easy_install_path``` in develop either deletes or overwrites the easy_install.pth file (since it opens in "w" mode). If it was accidentally linked in before then this would mitigate that, although it shouldn't have to.